chain/chain/src/chain.rs

```diff
@@ -119,6 +125,12 @@ impl BlockLike for Orphan {
     }
 }
 
+impl Orphan {
+    fn prev_hash(&self) -> CryptoHash {
+        *self.block.header().prev_hash()
```

Comment:
I think it is usually better to return a reference and let caller decide whether they want to clone/deference